features/registry/registry.feature

```diff
@@ -106,3 +106,87 @@ Feature: Testing registry
     And the "myimage2" image stream was created
     And the "myimage1" image stream becomes ready
     And the "myimage2" image stream becomes ready
+    
+  # @author wewang@redhat.com
+  # @case_id OCP-23030
+  @admin
+  Scenario: Enable must-gather object refs in image-registry cluster
+    When I run the :get admin command with:
+      | resource      | co             |
+      | resource_name | image-registry |
+      | o             | yaml           |
+    Then the step should succeed
+    And the output should contain:
+      | name: system:registry                      |
+      | resource: clusterroles                     |
+      | name: registry-registry-role               |
+      | resource: clusterrolebindings              |
+      | name: registry                             |       
+      | resource: serviceaccounts                  |
+      | name: image-registry-certificates          |
+      | resource: configmaps                       |
+      | name: image-registry-private-configuration |
+      | resource: secrets                          | 
+      | name: image-registry                       |
+      | resource: services                         |
+      | name: node-ca                              |
+      | resource: daemonsets                       |
+      | name: image-registry                       |
+      | resource: deployments                      |
+    When I run the :get admin command with:
```

Comment:
@wewang58 try to replace all the raw `oc :get` calls with object get.

Reply:
@pruan-rht you mean using "I get project xxx" step right? but some resources are cluster resources, do we have steps for these?